### PR TITLE
cleanup: schema generate function

### PR DESCRIPTION
**Pull request by rahulreddy**
This method makes it easier to manage key scheme for the data

### CODE REVIEW

**Comment by rahulreddy** [comment]
https://github.com/scality/S3/pull/151 to make sure S3 is ok

**Comment by LaurenSpiegel** [comment]
👍

**Comment by RachedBenMustapha** [comment]
👍

File: lib/schema.js [hunk 1]
```diff
@@ -1,243 +1,47 @@
...
-*/
-export function bucketNumberOfObjectsCounter(bucket) {
-    return `s3:buckets:${bucket}:numberOfObjects:counter`;
+* Returns the metric key in schema for the bucket
+* @param {string} bucket - bucket name
+* @param {string} metric - metric name
+* @return {string} - schema key
+*/
+export function genBucketKey(bucket, metric) {
+    const keys = {
```
Comment (alexandre-merle) [inline lib/schema.js]: moving this outside the function will be more efficient

**Comment by alexandre-merle** [comment]
one small comments, :+1:

**Comment by rahulreddy** [comment]
thanks updated that.

**Pull request merged**